test/index.js

```diff
@@ -40,3 +40,17 @@ describe("check if date is invalid", function () {
     DateValidator.isValidDate("AA/BB/2017", "DD/MM/YYYY").should.equal(false);
   });
 });
+
+describe("check if date is in the past or not", function() {
+  it("returns false for past check on given date of (2019-09-29, YYYY-MM-DD)", function () {
+    DateValidator.isDatePast("2019-09-29", "YYYY-MM-DD").should.equal(false);
```

Comment:
I think we should use moment for getting future date, so test does not fails even if its `2020`

Reply:
So I'm thinking of the `moment().add()` functionality. Is that what you're thinking of?

Reply:
yes. thats what I had in my mind